BGAnimations/ScreenGameplay underlay/PerPlayer/ErrorBar/Monochrome.lua

```diff
@@ -70,8 +70,15 @@ local af = Def.ActorFrame{
 
             tick:finishtweening()
 
+
+            local is_W0 = true
```

Comment:
Maybe rename this variable. At first it was a confusing for me why we set `is_W0 = true` for all the judgments.

Reply:
I replaced it entirely with just:

```lua
local color = judgmentColors[params.TapNoteScore] 

-- Check if we need to adjust the color for the white fantastic window.
if mods.ShowFaPlusWindow and ToEnumShortString(params.TapNoteScore) == "W1" and not IsW0Judgment(params, player) then
    color = SL.JudgmentColors["FA+"][2]
end
```